xl_auth/commands.py

```diff
@@ -608,3 +609,12 @@ def _get_manually_deleted_permissions():
                   % (permission.user.email, permission.collection.code, wipe_permissions))
             if wipe_permissions:
                 permission.delete()
+
+
+@click.command()
+def prod_run():
+    """Run application with production setup."""
+    from sh import Command
+    flask_cmd = Command('flask')
+    click.echo(flask_cmd('db', 'upgrade').stderr)
```

Comment:
@mxtthias After our talk about "why having a gunicorn_conf.py" I think I see your point. Letting this `flask prod_run` be a shortcut equivalent to running `gunicorn -c *onf.py autoapp:app` on the command line. Along the same line of reasoning, it seems logical to also have the app's database mgnt wrapped up in start/reload hooks. See #184 for a proposal on how to achieve this.